addCRH.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
```

Comment:
Oh, my mistake. The script wasn't intended to be added to your repo; I'll remove it from the branch again. However, feel free to use it.